pkg/server/backuprestoreserver.go

```diff
@@ -224,22 +224,22 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 			}
 			_ = miscellaneous.SleepWithContext(ctx, retryTimeout)
 		}
-	} else {
-		// when OriginalClusterSize = 1
-		m := member.NewMemberControl(b.config.EtcdConnectionConfig)
-		err := retry.OnError(retry.DefaultBackoff, func(err error) bool {
-			return err != nil
-		}, func() error {
-			cli, err := etcdutil.NewFactory(*b.config.EtcdConnectionConfig).NewCluster()
-			if err != nil {
-				return err
-			}
-			_, err = m.UpdateMemberPeerURL(ctx, cli)
-			return err
-		})
+	}
+
+	// when OriginalClusterSize = 1
+	m := member.NewMemberControl(b.config.EtcdConnectionConfig)
+	err := retry.OnError(retry.DefaultBackoff, func(err error) bool {
+		return err != nil
+	}, func() error {
+		cli, err := etcdutil.NewFactory(*b.config.EtcdConnectionConfig).NewCluster()
 		if err != nil {
-			b.logger.Error("unable to update the member")
+			return err
```

Comment:
can you please log the error `fmt.Errorf("unable to create etcd cluster client ",err)` as well , so we can get to know the reason behind if we see this error log `unable to update the member`

Reply:
i discussed this with Aaron. He did not want to return an error in this. Can you check with him once?

Reply:
This line in inside a `retry.OnError` function, so imo it's okay if it is not logged here as it will be anyway logged in line [239](https://github.com/gardener/etcd-backup-restore/pull/534/files#diff-df6f7abbd9069d0a874a18b5717de5b0fcd68dac32184fdcdc2a26019be2296fR239)
Having said that, can you please update [line 239](https://github.com/gardener/etcd-backup-restore/pull/534/files#diff-df6f7abbd9069d0a874a18b5717de5b0fcd68dac32184fdcdc2a26019be2296fR239) to print the error along with the log?

Reply:
done